ert_shared/cli.py

```diff
@@ -30,12 +35,27 @@ def run_cli(args):
         model, argument = _setup_ensemble_smoother(args)
     elif args.mode == 'es_mda':
         model, argument = _setup_multiple_data_assimilation(args)
+    
     else:
         raise NotImplementedError(
             "Run type not supported {}".format(args.mode))
 
     model.runSimulations(argument)
 
+def _execute_workflow(workflow_name):
+    logging.basicConfig(level=logging.INFO)
+    workflow_list = ERT.ert.getWorkflowList()
+    try:
+        workflow = workflow_list[workflow_name]
+    except KeyError:
+        logging.error("Workflow {} is not in the list of available workflows".format(workflow_name))
+        return
+    context = workflow_list.getContext()
+    workflow.run(ert=ERT.ert, verbose=True, context=context)
+    all_successfull = all([v['completed'] for k, v in workflow.getJobsReport().items()])
+    if all_successfull:
+        logging.info("Workflow {} ran successfully!".format(workflow_name))
```

Comment:
And if not all are successful, does the user get any feedback?

Reply:
yes, `workflow.run()` outputs error messages.